File: cs50x/1week/mario/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int size;
    do
    {
        size = get_int("Height: ");
    }
    while(size < 1 || size > 8);

    for(int height = 0; height < size; height++){
        for(int space = size; space > height+1; space--){
            printf(" ");
        }
        for(int tile = 0; tile <= height; tile++){
            printf("#");
        }
        printf("\n");
    }
}

File: cs50x/6week/mario/mario-more/mario.py
from cs50 import get_int

while True:
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break

for row in range(0, height):
    for space in range(height, row + 1, -1):
        print(" ", end="")

    for column in range(row + 1):
        print("#", end="")

    print("  ", end="")

    for column in range(row + 1):
        print("#", end="")

    print()